webpack.config.js

```diff
@@ -38,7 +38,9 @@ var commonConfig = {
       bootstrap: '$',
       bootstraptour: 'Tour',
       'jquery-ui': '$',
-      typeahead: '$.typeahead'
+      typeahead: '$.typeahead',
+      'codemirror/lib/codemirror': 'commonjs codemirror/lib/codemirror',
+      'codemirror/mode/meta': 'commonjs codemirror/mode/meta',
```

Comment:
Great ! Thanks can you post a link of where you found this info ? 
In a separate PR I guess it will be useful to do the same for `addon`, `keymap`, and `theme`.